app/controllers/search_controller.rb

```diff
@@ -28,8 +45,24 @@ def chat_channel_params
       channel_status
       status
     ]
+
+    params.permit(accessible)
+  end
+
+  def classified_listing_params
+    remove_blank_params
```

Comment:
Could this be a before_action that applies only to the classified_listings endpoint? Since that’s rails convention, I’d say we should do that unless there’s a specific reason not to!

Reply:
I’m thinking the action could be something like sanitize_params or the like